package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "nwHacks2020",
+  "name": "cmd-f2020",
   "version": "1.0.0",
-  "description": "nwHacks is the largest 24-hour hackathon in Western Canada, with more than 700 participants in 2019. Whether you're a seasoned hacker or a tech newbie, nwHacks welcomes you; just bring an open mind and an insatiable desire to learn, and we’ll take care of the rest.",
+  "description": "British Columbia’s all-female* hackathon. Held March 7-8th, 2020 at UBC",
```

Comment:
Was this copy given to dev?
I thought we call ourselves `Vancouver's all-female* hackathon`